packages/admin/cms-admin/src/pages/pageSearch/usePageSearch.ts

```diff
@@ -6,7 +6,7 @@ import { TextMatch } from "../../common/MarkedMatches";
 import { GQLPageSearchFragment } from "../../graphql.generated";
 import { PageTreePage } from "../pageTree/usePageTree";
 
-export type PageSearchMatch = TextMatch & { page: { id: string; ancestorIds: string[] } };
+export type PageSearchMatch = TextMatch & { page: { id: string; ancestorIds: string[] } } & { where?: "name" | "path" };
```

Comment:
```suggestion
export type PageSearchMatch = TextMatch & { page: { id: string; ancestorIds: string[] }, where: "name" | "path" };
```

I suppose `where` should be required.